src/get-started/java-binding.md

```diff
@@ -233,29 +223,40 @@ in the schema class that implements [`Schema`][schema] interface; when implement
        * "<service name>.test-map".
        */
       ProofMapIndexProxy<String, String> testMap() {
-        var fullName = fullIndexName("test-map");
-        return ProofMapIndexProxy.newInstance(fullName, view, string(),
-            string());
-      }
-
-      /** Creates a full index name given its simple name. */
-      private String fullIndexName(String name) {
-        return namespace + "." + name;
+        var address = IndexAddress.valueOf("test-map");
+        return view.getProofMap(address, string(), string());
       }
     }
     ```
 
+#### Blockchain Data
+
+[`BlockchainData`][blockchain-data] is the object providing access to blockchain
+data of a particular service instance.
+
+The service instance data is accessible via a `Prefixed` access which isolates
+the service data from all the other instances.
+
+On top of that, this class provides read-only access to persistent data of:
+
+- [Exonum Core][blockchain], containing information on blocks, transactions,
+  execution results, consensus configuration, etc.
+- Dispatcher, containing information on active service instances.
```

Comment:
Nit: Dispatcher schema also provides information on artifacts. Or is this not implemented in EJB?

Reply:
No, currently only a single index is exposed.

Reply:
I added the artifacts, if someone sees in the docs and then misses in the schema, they will let us know.